Image search bug

- (maybe ok?) empty query doesn't return any results
- (ok) query for non-existent match doesn't return results
- (bug) query for existing match is 500

Local log in backend when searching for "thumb" which is the title on an image or two

----
```
19:05:54 [ERROR] Internal Server Error: ""
19:05:54 [ERROR] ISE while responding to request: decode error: error decoding response body: invalid type: map, expected a string at line 1 column 2884

Caused by:
    0: error decoding response body: invalid type: map, expected a string at line 1 column 2884
    1: invalid type: map, expected a string at line 1 column 2884
19:05:54 [INFO] 127.0.0.1:59864 "GET /v1/image?q=thumb&page=0 HTTP/1.1" 500 0 "http://localhost:10001/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.75 Safari/537.36" 0.112323
```

--- a/backend/api/Cargo.toml
+++ b/backend/api/Cargo.toml
@@ -13,7 +13,7 @@ actix-cors = "0.3.0"
 actix-rt = "1.0.0"
 actix-service = "1.0.5"
 actix-web = "3.0.0"
-algolia = { git = "https://gitlab.com/izik1/algolia-rs.git", rev = "61911d09cbf0c4361c73afe64fdbd881cdbb0f95" }
+algolia = { git = "https://gitlab.com/izik1/algolia-rs.git", rev = "85975c32cc9396c3e4ae6df7158948579b9989f4" }
 anyhow = "1.0.32"
 chrono = "0.4.13"
 chrono-tz = "0.5.3"

